From 75efc124ac0aa45b6bd9637e1994ca7613910cf8 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Tue, 4 Aug 2015 15:51:54 +0800
Subject: [PATCH 0045/5242] MLK-11318-2 ARM: dts: imx7d-sdb: add dts support
 for pwm backlight

commit  facb0fa074f4b23ee9b7cf5b7ba03febeaa97577 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dts support for pwm backlight on imx_4.1.y branch.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 8256b20..857a223 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -129,6 +129,14 @@
 			};
 		};
 	};
+
+	backlight {
+		compatible = "pwm-backlight";
+		pwms = <&pwm1 0 5000000>;
+		brightness-levels = <0 4 8 16 32 64 128 255>;
+		default-brightness-level = <6>;
+		status = "okay";
+	};
 };
 
 &adc1 {
-- 
1.7.9.5

